app/views/good_job/batches/_table.erb

```diff
@@ -76,7 +76,7 @@
             </div>
           </div>
           <%= tag.div id: dom_id(batch, "properties"), class: "batch-properties list-group-item collapse small bg-dark text-light" do %>
-            <%= tag.pre JSON.pretty_generate(batch.properties) %>
+            <%= tag.pre JSON.pretty_generate(JSON.parse(batch.read_attribute_before_type_cast(:serialized_properties))) %>
```

Comment:
Could this more simply be `serialized_properties` directly because it will be typecast as JSON?

Reply:
hmm I think it doesn't work, the issue happens when the model is deserialized, so most of the methods actually raise an error.

Reproduction steps:
```ruby
record = GoodJob::Job.create # Or any ActiveRecord instance
batch = GoodJob::Batch.enqueue(record: record)
record.destroy!

batch.reload => # An error occurred when inspecting the object: #<ActiveJob::DeserializationError: Error while trying to deserialize arguments: Couldn't find GoodJob::Job with 'id'=20ea1655-c5ca-4cbc-988e-aae7a6b1dad8>

batch._record.id # => "some-uuid"
batch._record.serialized_properties # Error while trying to deserialize arguments: Couldn't find GoodJob::Job with 'id'=20ea1655-c5ca-4cbc-988e-aae7a6b1dad8 (ActiveJob::DeserializationError)
```

Reply:
My initial approach was to actually catch the deserialization error (see [first commit](https://github.com/bensheldon/good_job/pull/1628/commits/60c41f6ddab63972df5e027a5eedae3440c9e0c0)). As I mentioned in the description, I don't think it's the right way. It fixes the batch record though

Reply:
Can this mirror how Job's serialized_parameters are handled?

Reply:
I added a new `display_serialized_properties` method that handles the deserialization errors, and also updated the `display_attributes` with the same logic.

The Job model always keeps the `serialized_params` data unserialized, and it's actually serialized only when the instance is casted into an active_job. If that's what you meant, we should get ride of the serialization logic for `serialized_properties` and only keep it for the `properties` methods since that is the "public" interface. Can give it a try if you agree with that.